pkg/sidecar-controller/snapshot_controller.go

```diff
@@ -511,8 +527,6 @@ func (ctrl csiSnapshotSideCarController) removeContentFinalizer(content *crdv1.V
 	if err != nil {
 		klog.Errorf("failed to update content store %v", err)
 	}
-
-	klog.V(5).Infof("Removed protection finalizer from volume snapshot content %s", content.Name)
```

Comment:
Why moving this to the beginning when finalizer is not removed yet?

Reply:
right, will put it back. Merge issue I think